mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -355,13 +355,28 @@ class AFTSurvivalRegressionModel private[ml] (
   @Since("2.0.0")
   override def transform(dataset: Dataset[_]): DataFrame = {
     transformSchema(dataset.schema, logging = true)
-    val predictUDF = udf { features: Vector => predict(features) }
-    val predictQuantilesUDF = udf { features: Vector => predictQuantiles(features)}
+
+    var predictionColNames = Seq.empty[String]
```

Comment:
These checks seem OK to add for consistency, though I think it still wouldn't come up unless the user explicitly changed both.